jumanji/training/configs/env/macvrp.yaml

```
@@ -0,0 +1,28 @@
+name: macvrp
+registered_version: MACVRP6-v0
```

Can we not bound the number of customers to the name?
```suggestion
registered_version: MACVRP-v0
```

---

Updated:+1: